Formula/mailcatcher.rb

```diff
@@ -0,0 +1,176 @@
+class Mailcatcher < Formula
+  desc "Catches mail and serves it through a dream"
+  homepage "https://mailcatcher.me"
+  url "https://github.com/sj26/mailcatcher/archive/refs/tags/v0.8.1.tar.gz"
+  sha256 "c40105f63407efc3fb18c2b1e2408896e4b86b0c8ba4e53ea17e087cbb2d7fee"
+  license "MIT"
+
+  depends_on "pkg-config" => :build
+
+  uses_from_macos "curl" => :test
+  uses_from_macos "expect" => :test
+  uses_from_macos "netcat" => :test
+  uses_from_macos "libffi"
+  uses_from_macos "ruby"
+  uses_from_macos "sqlite"
+
+  on_linux do
+    depends_on "node" => :build
+  end
```

Comment:
Do we know why this needs `node` on Linux but not on macOS?

Reply:
The `rake assets` uses ExecJS to convert Coffeescript to JS.  ExecJS requires some JS engine, and can use the builtin JS engine in OSX.  Linux has no such thing.

Reply:
execjs is installed on build by `bundler install` it's not a run time dependancy.